ooniprobe/View/TestResults/Footer/UploadFooterViewController.m

```diff
@@ -0,0 +1,124 @@
+#import "UploadFooterViewController.h"
+#import "MessageUtility.h"
+#import <QuartzCore/QuartzCore.h>
+#import "SettingsUtility.h"
+#import <mkall/MKCollector.h>
+#import "TestUtility.h"
+#import "MBProgressHUD.h"
+
+@interface UploadFooterViewController ()
+
+@end
```

Comment:
This is the xcode generated file. I never touch it.

Reply:
I feel brave today :-). Can you please remove it? I'm sure I've written ObjectiveC code where this internal interface was not there, so we can safely remove it.

Reply:
Removing it still builds the project.
The fact is this interface is in basically *every* file so maybe is better to have a PR with a general cleanup rather than doing it only once in one file

Reply:
Let's start with not increasing the entropy in this PR. Refactoring other files can be done incrementally.

("Always leave the camp cleaner than you find it.")

Reply:
so do you think this can be refactored here or in a next PR?

Reply:
Just remove the three lines in this file, so we don't ever commit them. We'll deal with the rest with time and calm as there is no rush to convert all the code base today, just the desired to make this PR minimal.